interface/billing/billing_report.php

```diff
@@ -2,25 +2,25 @@
 /**
```

Comment:
TODO:
Flagged this file to look at history in detail

Reply:
Looking into it and working on it  . I had made the initial change on 11/12/2017. Incorporated the  changes from Nov 25 commit. Also added the <?php echo xlt('UB04 Margins'); ?>: code.

I know that using anchor tags only for actions is discouraged,  however I find the implementation of the actions to be selected as buttons makes it look a bit messy and jumbled up. I therfore left it as an anchor tag.

Billing manager in demo:

![billing_manager_master-openemr](https://user-images.githubusercontent.com/12654295/38978030-00743a90-436b-11e8-9f2a-496caf5fc826.png)

Billing manager in this PR:

![billing_manager_zbig01-openemr](https://user-images.githubusercontent.com/12654295/38978038-06910264-436b-11e8-82fb-1ec048ff5e5e.png)